# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436890803 14400
#      Tue Jul 14 12:20:03 2015 -0400
# Node ID 86299d2fbf87cd2faa9013d7bf983adf4f875bea
# Parent  10f67e9d1e70e4eb23af61608a9a7eb7cb922279
* libinterp/module.mk: Don't add BUILT_DISTFILES to MAINTAINERCLEANFILES.

diff --git a/libinterp/module.mk b/libinterp/module.mk
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -53,26 +53,26 @@ ULT_PARSER_SRC := \
   libinterp/corefcn/oct-tex-parser.in.yy \
   libinterp/parse-tree/oct-parse.in.yy
 
 GENERATED_PARSER_FILES := \
   libinterp/corefcn/oct-tex-lexer.ll \
   libinterp/corefcn/oct-tex-parser.yy \
   libinterp/parse-tree/oct-parse.yy
 
-BUILT_DISTFILES += \
+LIBINTERP_BUILT_DISTFILES = \
   $(GENERATED_PARSER_FILES) \
   libinterp/corefcn/oct-tex-parser.h \
   libinterp/corefcn/oct-tex-symbols.cc \
   libinterp/parse-tree/oct-gperf.h \
   libinterp/parse-tree/oct-parse.h
 
 ## Files that are created during build process and installed,
 ## BUT not distributed in tarball.
-BUILT_NODISTFILES += \
+LIBINTERP_BUILT_NODISTFILES = \
   libinterp/corefcn/mxarray.h \
   libinterp/corefcn/oct-errno.cc \
   libinterp/corefcn/defaults.h \
   libinterp/corefcn/graphics.h \
   libinterp/builtin-defun-decls.h \
   libinterp/operators/ops.cc \
   libinterp/oct-conf.h \
   libinterp/oct-conf-features.h \
@@ -90,17 +90,17 @@ EXTRA_DIST += \
   libinterp/genprops.awk \
   libinterp/mk-errno-list \
   libinterp/mk-pkg-add \
   libinterp/mkbuiltins \
   libinterp/mkdefs \
   libinterp/mkops \
   libinterp/oct-conf.in.h \
   libinterp/version.in.h \
-  $(BUILT_DISTFILES)
+  $(LIBINTERP_BUILT_DISTFILES)
 
 octinclude_HEADERS += \
   libinterp/corefcn/graphics-props.cc \
   libinterp/parse-tree/oct-gperf.h \
   libinterp/builtins.h \
   libinterp/builtin-defun-decls.h \
   libinterp/octave.h \
   libinterp/options-usage.h \
@@ -362,15 +362,17 @@ CLEANFILES += \
   $(DLDFCN_PKG_ADD_FILE) \
   libinterp/corefcn/graphics-props.cc \
   libinterp/corefcn/oct-tex-parser.output \
   libinterp/parse-tree/oct-parse.output
 
 DISTCLEANFILES += \
   libinterp/.DOCSTRINGS \
   libinterp/DOCSTRINGS \
-  $(BUILT_NODISTFILES) \
+  $(LIBINTERP_BUILT_NODISTFILES) \
   $(OCT_FILES) \
   $(LIBINTERP_TST_FILES)
 
 MAINTAINERCLEANFILES += \
-  $(BUILT_DISTFILES)
+  $(LIBINTERP_BUILT_DISTFILES)
 
+BUILT_DISTFILES += $(LIBINTERP_BUILT_DISTFILES)
+BUILT_NODISTFILES += $(LIBINTERP_BUILT_NODISTFILES)
